src/components/MediaObject/MediaObject.md

```diff
@@ -102,6 +102,64 @@ const avatar = (size) => (
 </div>
 ```
 
+With ellipsis:
+
+```js { "props": { "data-example": "with ellipsis" } }
+const { AvatarSize } = require('../Avatar');
+const { MediaObjectSize } = require('.');
+
+const name = 'Hubert Blaine Rindfleischetikettierungsüberwachungsaufgabenübertragungsgesetz';
+const title = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt';
+const status = 'Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque';
+
+const avatar = (size) => (
+  <Avatar imageUrl={user.imageUrl} name={user.name} size={size} />
+);
+
+<div>
+  <MediaObject
```

Comment:
we probably don't need to show the example at every size, just one size should be fine to show the text wrapping